AdobeStockImageAdminUi/Test/Mftf/Suite/AdobeStockIntegrationEnhancedMediaGallerySuite.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<suites xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Suite/etc/suiteSchema.xsd">
+    <suite name="AdobeStockIntegrationEnhancedMediaGallerySuite">
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminDisableWYSIWYGActionGroup" stepKey="disableWYSIWYG" />
+            <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="setCorrectModuleConfig"/>
+            <actionGroup ref="AdminMediaGalleryEnhancedEnableActionGroup" stepKey="enableEnhancedMediaGallery">
+                <argument name="enabled" value="1"/>
+            </actionGroup>
+        </before>
+        <after>
+            <actionGroup ref="AdminEnableWYSIWYGActionGroup" stepKey="enableWYSIWYG" />
```

Comment:
I think we need to add AdminLogout action in after block

Reply:
Hi @Nazar65, none of the others Suites implement an admin logout on after block, I believe each test starts with a clear session, but this Suite was deleted anyway, thanks for your comment.

Reply:
> To wrap up: The lack of adminLogoutActionGroup in after section does not affect other tests, however it's considered to be best practice, as (!) you should follow the scout rule - leave the place better than you found when you started.